Docs Install: Added preview downloads and fixed formatting (#858)
files-community/Website: src/routes/docs/getting-started/install/+page.md

---

<script>
  import { Button, InfoBar } from "fluent-svelte";
  import { InfoBar } from "fluent-svelte";
</script>

## Install and get started with Files

<Button href="/download/" variant="accent">
	Install Files
</Button>

Files is a community-driven project that depends on your support to grow and improve. Please consider purchasing Files through the [Microsoft Store](ms-windows-store://pdp/?ProductId=9nghp3dx8hdx&cid=FilesWebsite) or supporting us on [GitHub](https://github.com/sponsors/yair100) if you install the free version.

You can also use the [preview version](ms-windows-store://pdp/?ProductId=9NSQD9PKV3SS&cid=FilesWebsite) alongside the stable release to get early access to new features and improvements.

<InfoBar title="Note" severity="information">

> 	The Windows Update and Delivery Optimization services will need to be enabled in order for the installation to work. Modified versions of Windows or running "debloat" scripts/programs are often missing these components and aren't supported.

Reviewer: On second thought, it might be helpful to have this info upfront instead of hoping the user will check the troubleshoot section.

Author: I added it back along with the check troubleshooting message

> 	The Windows Update and Delivery Optimization services will need to be enabled in order for the installation to work. Modified versions of Windows or running "debloat" scripts/programs are often missing these components and aren't supported. Check the troubleshooting section at the bottom if you have issues installing.

Reviewer: ```suggestion
	The Windows Update and Delivery Optimization services will need to be enabled in order for the installation to work. Modified versions of Windows or running "debloat" scripts/programs are often missing these components and aren't supported. Check the troubleshooting section below for further details.
```
